src/assets/images/logo-dark.svg

```diff
@@ -0,0 +1,54 @@
+    </g>
+  </g>
```

Comment:
Please confirm that this is valid since GitHub is marking this in red, but I don't know why...

Reply:
Yes. Sometimes GH marks such things. Don't know exactly where this is coming from. Maybe a wrong line ending ...